app/views/includes/head.jade

```diff
@@ -7,7 +7,7 @@ head
   meta(http-equiv='Content-type', content='text/html;charset=UTF-8')
   meta(name="keywords", content="cards against humanity, cards for humanity, offensive card game")
   meta(name="description", content="Cards for Humanity: A Game for Horrible People Desperately Trying to do Good.")
-
+  meta(name="viewport" content="width=device-width, initial-scale=1.0")
   meta(name="apple-mobile-web-app-capable", content="yes")
   meta(name="viewport", content="initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=0")
```

Comment:
Do we need to duplicate what you have on line 10? Please confirm that line 12 is already doing what you need.